.licenserc.yaml

```diff
@@ -77,6 +77,8 @@ header: # `header` section is configurations for source codes license header.
     - '**/*.pb.go'
     - '**/*.textproto'
     - '**/*.pem'
+    - '**/.gitignore'
+    - '.gitmodules'
```

Comment:
I think we don't use submodule in go project?

Reply:
Fixed